gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/MysqlUserQuotaManager.java

```diff
@@ -62,7 +80,7 @@ public void init(Collection<Dag<JobExecutionPlan>> dags) {
   @Override
   int incrementJobCount(String user, CountType countType) throws IOException {
     try {
-      return this.mysqlStore.increaseCount(user, countType);
+      return this.quotaStore.increaseCount(user, countType);
```

Comment:
Question here, shouldn't we change the increaseCount and addDagId to be one mysql transaction? Otherwise we will see discrepancy between these two table?

Reply:
Probably yes, but it was not being done together in InMemory version also.
Do you want me to change both?